Index: src/test/java/tests/KaterynaSpecialOfferBannerTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package tests;\n\nimport core.BaseSeleniumTest;\nimport org.itfriendly.projectPages.MainPageBody;\nimport org.testng.Assert;\nimport org.testng.annotations.Test;\n\npublic class KaterynaSpecialOfferBannerTest extends BaseSeleniumTest {\n\n\n    @Test\n    public void clickOnBannersArrowtest() {\n        MainPageBody offerBanner = new MainPageBody();\n        offerBanner.openPage();\n        offerBanner.clickOnArrow();\n        Assert.assertTrue(offerBanner.isNextBannerArrowClickable());\n\n    }\n\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/tests/KaterynaSpecialOfferBannerTest.java b/src/test/java/tests/KaterynaSpecialOfferBannerTest.java
--- a/src/test/java/tests/KaterynaSpecialOfferBannerTest.java	(revision 7b109569b4570d2b63716c81263d2dcbad0e7154)
+++ b/src/test/java/tests/KaterynaSpecialOfferBannerTest.java	(date 1709118616963)
@@ -16,5 +16,6 @@
         Assert.assertTrue(offerBanner.isNextBannerArrowClickable());
 
     }
+    
 
 }
